xarray/tests/test_backends.py

```diff
@@ -1735,21 +1753,27 @@ def test_write_inconsistent_chunks(self) -> None:
 class ZarrBase(CFEncodedBase):
 
     DIMENSION_KEY = "_ARRAY_DIMENSIONS"
+    zarr_version = 2
```

Comment:
`zarr_version` is here to support conditional skipping in parameterized tests. We can remove this once we have feature parity between the spec versions.